src/features/board/Board.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { board, attack } from './boardSlice';
+import {selectCard, selectedCard} from '../card/selectedCardSlice';
 import { useAppSelector, useAppDispatch } from '../../app/hooks';
 import { CardInterface, Card } from '../card/Card';
+import { useSelector } from 'react-redux';
 
 export function Board () {
     const playerCards: CardInterface[] = useAppSelector(board).playerSide.cards;
     const opponentCards: CardInterface[] = useAppSelector(board).opponentSide.cards;
     const dispatch = useAppDispatch();
+    const selectedCardPosition = useSelector(selectedCard);
+
+    useEffect(() => {
+        window.addEventListener('mouseup', handleDeselectCard);
+        return () => {
+            window.removeEventListener('mouseup', handleDeselectCard);
+        };
+    }, [selectedCardPosition]);
+
+    const handleSelectCard = (cardPosition:number) => {
+        if (selectedCardPosition === null) {
+            dispatch(selectCard(cardPosition))
+        }
+    }
+
+    const handleDeselectCard = (event:MouseEvent) => {
+        if (event.which === 3) {
+            event.preventDefault();
+            dispatch(selectCard(null));
```

Comment:
I think we should keep two different actions selectCard and unselectCard because : 
- I think this is more explicit
- No parameters === less complexity
- The payloadAction type is simpler (no union with null)